sway/input/cursor.c

```diff
@@ -1234,6 +1234,11 @@ static void warp_to_constraint_cursor_hint(struct sway_cursor *cursor) {
 		double ly = sy + con->content_y - view->geometry.y;
 
 		wlr_cursor_warp(cursor->cursor, NULL, lx, ly);
```

Comment:
I'd expect `wlr_cursor_warp` to call `wlr_seat_pointer_notify_motion` and update the seat pointer state. Is this not the case?

Reply:
No, `wlr_cursor_warp` and friends only update the position of the cursor image. For instance, they're also used by tablet input, where sending pointer events would be incorrect.

Reply:
Ah, I see.